ui/revenuecatui/build.gradle.kts

```diff
@@ -65,6 +65,13 @@ android {
     }
 }
 
+tasks.withType<Test> {
+    // This is needed because of an incompatibility between Robolectric and Paparazzi:
+    // https://github.com/cashapp/paparazzi/issues/1979
+    forkEvery = 1
+    maxParallelForks = 5
```

Comment:
Honestly I'm not in love with this and it does cause tests to be noticeable slower... but the alternative is to split the Paparazzi tests (right now we have 2 classes) into a separate module, and we're using internal APIs, so that isn't straightforward.